qa/workunits/rbd/cli_migration.sh

```diff
@@ -124,8 +128,56 @@ EOF
 
     rbd migration abort ${dest_image}
 
+    # no snap name or snap id
```

Comment:
Nice that we've these tests now. Bugs in this part of the parsing code were missed earlier.

Reply:
> Bugs in this part of the parsing code were missed earlier.

That is why I added them ;)